tools/config.c

```diff
@@ -92,21 +92,26 @@ config_pin_minlen(char *path, const char *pinlen)
 {
 	fido_dev_t *dev;
 	char *pin = NULL;
-	int len, r, ok = 1;
+	int v, len, r, ok = 1;
 
 	dev = open_dev(path);
 	if ((len = base10(pinlen)) < 0 || len > 63) {
 		warnx("%s: len > 63", __func__);
 		goto out;
 	}
-	if ((r = fido_dev_set_pin_minlen(dev, (size_t)len, NULL)) != FIDO_OK &&
-	    should_retry_with_pin(dev, r)) {
-		if ((pin = get_pin(path)) == NULL)
+	if ((get_devopt(dev, "uvAcfg", &v) == 0) && (v > 0)) {
+		if ((r = fido_dev_set_pin_minlen(dev, (size_t)len, NULL)) != FIDO_OK &&
```

Comment:
If the device supports uvAcfg, wont this first set the min length using built-in uv, then prompt the user for a PIN and set the min length again?

Reply:
ohh right, that scenario :)